x-pack/plugins/secops/server/graphql/ecs/resolvers.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { GraphQLScalarType, Kind } from 'graphql';
+import { isBoolean, isNumber, isObject } from 'lodash/fp';
+
+/*
+ *  serialize: gets invoked when serializing the result to send it back to a client.
+ *
+ *  parseValue: gets invoked to parse client input that was passed through variables.
+ *
+ *  parseLiteral: gets invoked to parse client input that was passed inline in the query.
+ */
+
+export const toStringArrayScalar = new GraphQLScalarType({
```

Comment:
It looks like I'm seeing some of these functions duplicated in the file:

```ts
x-pack/plugins/secops/server/graphql/events/resolvers.ts
```

Can we consolidate to just one which has the tests with it?

Reply:
oh that's a mistake